components/ui-api-layer/internal/domain/k8s/resourcequotastatus_resolver_test.go

```diff
@@ -0,0 +1,83 @@
+package k8s
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/kyma-project/kyma/components/ui-api-layer/internal/domain/k8s/automock"
+	testingUtils "github.com/kyma-project/kyma/components/ui-api-layer/internal/testing"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"k8s.io/api/core/v1"
+	"k8s.io/client-go/informers"
+	"k8s.io/client-go/kubernetes/fake"
+)
+
+func TestResourceQuotaStatusResolver_ResourceQuotaStatus_HappyPath(t *testing.T) {
```

Comment:
please change this tests. You should only mock the Service nothing more.
You should have two tests:
 - happy path (error = nil)
 - failure (error != nil )